src/codec/video/frame.rs

```diff
@@ -382,6 +419,17 @@ impl VideoFrameMut {
         self
     }
 
+    /// Get picture type
+    pub fn picture_type(&self) -> PictureType {
+        unsafe { PictureType::from_i32(ffw_frame_get_picture_type(self.ptr)).unwrap() }
+    }
+
+    /// Set picture type
+    pub fn with_picture_type(self, picture_type: PictureType) -> Self {
+        unsafe { ffw_frame_set_picture_type(self.ptr, picture_type as i32) };
```

Comment:
Do not confuse `c_int` and `i32`. They are not necessarily the same thing on all platforms.

```suggestion
        unsafe { ffw_frame_set_picture_type(self.ptr, picture_type as c_int) };
```